frontend/my-app/src/requestObjects.ts

```diff
@@ -3,17 +3,23 @@ enum HttpMethods {
   POST = 'POST',
 }
 
+const ENDPOINT = process.env.NODE_ENV === 'development' ? '' : 'https://b378-205-175-97-210.ngrok.io'
```

Comment:
This is probably the kind of thing we want to store in an environmental variable

Reply:
if you add `.env` folder you can define them there, here's a helpful link https://create-react-app.dev/docs/adding-custom-environment-variables/